app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -278,6 +279,22 @@ def volumes
     end
   end
 
+  # Store VPC volume profiles in cloud_volume_types table. Use name for the ems_ref and name. Use the family as description.
+  # @return [void]
+  def cloud_volume_types
+    $ibm_cloud_log.info('Starting cloud volume types persistance.')
+    collector.volume_profiles.each do |v|
+      $ibm_cloud_log.info("Saving #{v} to persister.")
+      persister.cloud_volume_types.build(
+        :type        => "ManageIQ::Providers::IbmCloud::VPC::StorageManager::CloudVolumeType",
```

Comment:
You shouldn't have to explicitly set the type here, this is done by the persister auto_model_class

Reply:
Ok, I was copying the Power Teams implementation. You might want to add a task for them to remove types from their parser.

Reply:
:+1:

Reply:
cc @jaywcarman ^